owa/app/js/components/manage_autogenration_option/ManageAutogenerationOption.jsx

```diff
@@ -0,0 +1,452 @@
+import React, { Component } from 'react';
+import { Button, Container, Row, Col, FormGroup, Label, Input, Table,
+    Modal, ModalHeader, ModalBody, ModalFooter, Alert } from 'reactstrap';
+import { MdDelete } from 'react-icons/lib/md';
+import './css/manage-autogeneration.css';
+
+class ManageAutogenerationOption extends Component {
```

Comment:
To be consistent, i would name this ManageAutoGenerationOption

Reply:
Done!